sdk/core/azure-core/src/azure_assert.cpp

```diff
@@ -4,12 +4,10 @@
 #include "azure/core/azure_assert.hpp"
 
 // Calling this function would terminate program, therefore this function can't be covered in tests.
-// LCOV_EXCL_START
```

Comment:
It could be covered in a special test case that wraps/handles/expects the process to terminate.  Not sure building out that infrastructure is warranted.